#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "first_project"
gen = ParameterGenerator()

gen.add("unsafety_threshold",    int_t,    0, "under this value, the status flag will be UNSAFE", 5,  5, 50)
gen.add("crash_threshold", double_t, 1, "under this value, the status flag will be CRASH", 0.5, 1,   5)

exit(gen.generate(PACKAGE, "publisher", "parameters"))
